src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionCoordinator.swift

```diff
@@ -13,7 +13,8 @@ final class ExposureDetectionCoordinator {
 	private let homeState: HomeState
 	private let exposureManager: ExposureManager
 	private let otpService: OTPServiceProviding
-	
+	private let surveyURLProvider: SurveyURLProvidable
+
```

Comment:
maybe change SurveyURLProvidable -> SurveyURLProviding to be consistent with the namings across the project

Reply:
done